stable/dex/templates/certificate.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if  (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1") }}
+apiVersion: certmanager.k8s.io/v1
```

Comment:
Looks like you need this change: https://github.com/mesosphere/charts/pull/803/files#r496285285